Orthtree/examples/Orthtree/orthtree_build.cpp

```diff
@@ -1,27 +1,27 @@
 #include <iostream>
 
 #include <CGAL/Epick_d.h>
+#include <CGAL/Cartesian_d.h>
 #include <CGAL/Orthtree.h>
-#include <CGAL/Orthtree_traits_d.h>
+#include <CGAL/Orthtree_traits_point.h>
 #include <CGAL/Random.h>
 
 // Type Declarations
-typedef CGAL::Dimension_tag<4> Dimension;
-typedef CGAL::Epick_d<Dimension> Kernel;
-typedef Kernel::Point_d Point_d;
-typedef std::vector<Point_d> Point_vector;
-
-typedef CGAL::Orthtree_traits_d<Kernel, Dimension> Traits;
-typedef CGAL::Orthtree<Traits, Point_vector> Orthtree;
+const int dimension = 4;
+using Kernel = CGAL::Epick_d<CGAL::Dimension_tag<dimension> >;
+using Point_d = Kernel::Point_d;
+using Point_vector = std::vector<Point_d>;
+using Traits = CGAL::Orthtree_traits_point<Kernel, Point_vector>;
+using Orthtree = CGAL::Orthtree<Traits>;
 
 int main()
 {
   CGAL::Random r;
 
   Point_vector points_dd;
-  for (std::size_t i = 0; i < 5; ++ i)
+  for (std::size_t i = 0; i < 500; ++ i)
```

Comment:
You raised it from 5 to 500, but that are too many to see them written to `cout`.  Would 20  do it to illustrate something?